x-pack/test/api_integration/apis/cloud_security_posture/update_rules_config.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import Chance from 'chance';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+  const chance = new Chance();
+
+  describe('POST /internal/cloud_security_posture/update_rules_config', () => {
+    let agentPolicyId: string;
+
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/empty_kibana');
+      await esArchiver.load('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/empty_kibana');
+      await esArchiver.unload('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
+    });
+
+    before(async function () {
+      const { body: agentPolicyResponse } = await supertest
+        .post(`/api/fleet/agent_policies`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          name: 'Test policy',
+          namespace: 'default',
+        });
+      agentPolicyId = agentPolicyResponse.item.id;
+    });
+
+    it(`should return 500 when package policy id not exist`, async () => {
+      const packagePolicyId = chance.guid();
+
+      const { body: response } = await supertest
+        .post(`/internal/cloud_security_posture/update_rules_config`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          package_policy_id: packagePolicyId,
+        })
+        .expect(500);
+      expect(response.error).to.be('Internal Server Error');
+      expect(response.message).to.be(`package policy Id '${packagePolicyId}' is not exist`);
```

Comment:
Let's fix the error message
```suggestion
      expect(response.message).to.be(`Package policy Id '${packagePolicyId}' does not exist`);
```